src/sonarwhal-theme/helper/index.js

```diff
@@ -242,6 +242,20 @@ module.exports = function () {
         getSortedToCTitles: (pages) => {
             return Object.keys(pages).sort();
         },
+        getTargetPagePath: (page, target) => {
+            const { category, permalink } = page;
```

Comment:
Can you explain what is going on in here? It will be good to have some examples of inputs and outputs.